Vizzuality/heco-invest
# Update the priority landscapes section of the For Investors page

This PR updates the priority landscapes section of the For Investors page to help users locate them on the map.

## Testing instructions

- Make sure the layout of the For Investors page corresponds to the design
- Make sure the layout of the For PDs page is not broken
- Make sure both pages are responsive
- Make sure that when hovering a priority landscape, the map is updated

## Tracking

[LET-1044](https://vizzuality.atlassian.net/browse/LET-1044)
